vm/actor/src/util/set_multimap.rs

```diff
@@ -114,11 +116,11 @@ where
         };
 
         set.for_each(|k| {
-            let v = parse_uint_key(&k)
+            let v = parse_usize_key(&k)
```

Comment:
cases like this is confusing why the use of usize key, because it's used as a u64

Reply:
oh whoops